tests/e2e-cypress/integration/mouseposition.cy.js

```diff
@@ -66,241 +65,167 @@ function checkMousePositionNumberValue(expectedX, expectedY, parser) {
 }
 
-describe('Test mouse position', () => {
-    context('Position in footer', () => {
-        beforeEach(() => {
-            cy.viewport('ipad-2')
-            cy.goToMapView({
-                center: proj4(WGS84.epsg, DEFAULT_PROJECTION.epsg, defaultCenter),
-                z: 12,
-            })
-            // waiting for OL to be loaded before testing
-            cy.get('[data-cy="ol-map"]').should('be.visible')
-        })
-        it('Shows LV95 coordinates by default', () => {
-            checkMousePositionNumberValue(2604624.64, 1261029.16, parseLV)
-        })
-        it('switches to LV03 when this SRS is selected in the UI', () => {
-            getMousePositionAndSelect(LV03Format)
-            checkMousePositionNumberValue(604624.6, 261029.21, parseLV)
-        })
-        it('switches to MGRS when this SRS is selected in the UI', () => {
-            getMousePositionAndSelect(MGRSFormat)
-            checkMousePositionStringValue('32TLT 87030 61820')
-        })
-        it('switches to WGS84 when this SRS is selected in the UI', () => {
-            getMousePositionAndSelect(WGS84Format)
-            checkMousePositionStringValue(`47° 30′ N 7° 30′ E (47.50000, 7.50000)`)
-        })
-        it('goes back to LV95 display if selected again', () => {
-            // Change display projection without moving the mouse
-            getMousePositionAndSelect(MGRSFormat)
-            getMousePositionAndSelect(LV95Format)
-            checkMousePositionNumberValue(2604624.64, 1261029.16, parseLV)
+describe('Test mouse position and interactions', () => {
+    const center = DEFAULT_PROJECTION.bounds.center.map((val) => val + 1000)
+    const centerLV95 = proj4(DEFAULT_PROJECTION.epsg, LV95.epsg, center)
+    const centerLV03 = proj4(DEFAULT_PROJECTION.epsg, LV03.epsg, center)
+    const centerWGS84 = proj4(DEFAULT_PROJECTION.epsg, WGS84.epsg, center)
+    const centerMGRS = MGRSFormat.format(center, DEFAULT_PROJECTION)
+    const centerUTM = UTMFormat.format(center, DEFAULT_PROJECTION)
+
+    beforeEach(() => {
+        cy.goToMapView({
+            center: center.join(','),
+            z: DEFAULT_PROJECTION.getDefaultZoom() + 3,
         })
     })
-    context('Fullscreen trigger', function () {
-        it('Activates fullscreen when nothing is under the cursor', () => {
-            cy.goToMapView()
-            cy.activateFullscreen()
-            cy.readStoreValue('getters.isPhoneMode').then((isPhoneMode) => {
-                if (isPhoneMode) {
-                    cy.get('[data-cy="map"]').click('center')
-                    cy.waitUntilState((state) => !state.ui.fullscreenMode)
-                }
-            })
-        })
+    it('shows coordinate under the mouse cursor in the footer, according to the selected projection format', () => {
+        // the viewport needs to be at least tablet sized in order for the mouse cursor tracker to be visible
+        cy.viewport('ipad-2')
```

Comment:
Here and below when using the `cy.viewport('iphone-4)` I would use the mocha `skip()` functionality and only run the tests if the viewport is set to the correct view. Until now we have all tests that should work on all viewport and the ability to run separately all view port. I would keep this strategy but if some test can only be run on a specific viewport, then we should only run it on that viewport and not force the viewport. So a proposal here would be to put all viewport specific test together in a context as such

```javascript
context('Iphone 4 tests', () => {
  before(() => {
     if (viewport != 'iphone-4') {
       this.skip()
     }
  })
  beforeEach(() => {
     cy.goToMapVeiw()
  })
  it('', () => {})
})
```